website/docs/using-plugins.md

```diff
@@ -512,6 +512,31 @@ module.exports = {
 };
 ```
 
+### `@docusaurus/plugin-baidu-analytics`
+
+It is a JavaScript tagging framework and API that allows you to send event data to Baidu Analytics.
+
+**Installation**
+
+```bash npm2yarn
+npm install --save @docusaurus/plugin-baidu-analytics
+```
+
+**Configuration**
+
+```js title="docusaurus.config.js"
+module.exports = {
+  plugins: ['@docusaurus/plugin-baidu-analytics'],
+  themeConfig: {
+    gtag: {
```

Comment:
that does not look good to have gtag here

Reply:
my mistake, it should be like this.
```
module.exports = {
  plugins: [
      [ '@docusaurus/plugin-baidu-analytics',{ 
        trackingID: 'xxxxxxxxx',
        // Optional fields.
        trackingDEV: true, // Should insert tracking script in development? default to false} 
      ]
     ],
}
```
and thanks for your reply, I`d like to publish the plugin under my own name, and add it to community plugins doc.

Reply:
great thanks

this triggered a discussion about a potential docusaurus org, will get back to you if a decision is made